src/main/scala/org/renci/robocord/RoboCORD.scala

```diff
@@ -100,61 +82,86 @@ object RoboCORD extends App with LazyLogging {
   val articlesTotal = metadata.size
   logger.info(s"Selected $articlesTotal articles for processing (from $startIndex until $endIndex, chunk $currentChunk out of $totalChunks)")
 
-  // We primarily pull the metadata from allMetadata, which includes abstracts. In some cases, however,
-  // we also have full text for those files.
+  // Identify full text SHAs and PMCIDs in the current chunk (`metadata`).
   val shasToLoad = metadata.map(_.get("sha")).flatten.flatMap(_.split(';')).map(_.trim.toLowerCase).toSet
-  val fullTextData: Seq[CORDArticleWrapper] = conf.data().flatMap(CORDJsonReader.wrapFileOrDir(_, logger, shasToLoad))
-  val fullTextBySHA: Map[String, Seq[CORDArticleWrapper]] = fullTextData.groupBy(_.sha1)
-  logger.info(s"Loaded ${fullTextData.size} full text documents corresponding to ${fullTextBySHA.keySet.size} SHA hashes.")
+  val pmcIdsToLoad = metadata.map(_.get("pmcid")).flatten.flatMap(_.split(';')).map(_.trim.toLowerCase).toSet
```

Comment:
Can `.map(_.get("pmcid")).flatten` be changed to `.flatMap(_.get("pmcid"))`?

Reply:
Fixed in 032cbbef94.